openforcefield/typing/chemistry/environment.py

```diff
@@ -18,6 +18,9 @@
 and David Mobley, UC Irvine.
 
 """
+
+# QUESTION: should we ditch camelCase and go with PEP8 standard snake_case?
```

Comment:
SGTM to do eventually; ambivalent about whether it is now.

Reply:
This is a carry over from @jchodera since I took the chemenv from the Topology branch. I think the answer to this is yes eventually, but not happening right now.